src/release_on_push_action/core.clj

```diff
@@ -42,7 +42,9 @@
                              (if (input-strategy-set?)
                                (getenv-or-throw "INPUT_STRATEGY")
                                (throw ex)))))
-   :dry-run             (contains? (set args) "--dry-run")})
+   :dry-run             (contains? (set args) "--dry-run")
+   :input/max-commits   (System/getenv "INPUT_MAX_COMMITS")
```

Comment:
We should move this up to `:input/tag-prefix` and also perform upfront validation via

`(Integer/parseInt (getenv-or-throw "INPUT_MAX_COMMITS"))`